lib/Cake/Controller/Controller.php

```diff
@@ -785,6 +785,9 @@ public function redirect($url, $status = null, $exit = true) {
 			}
 		}
 
+		if ($status === null) {
+			$status = 302;
+		}
 		if ($status) {
```

Comment:
I just wanted to keep it as BC as possible, we can maybe safely remove that check in 2.7?